atlasdb-config/build.gradle

```diff
@@ -37,3 +46,17 @@ dependencies {
     testCompile group: 'com.github.tomakehurst', name: 'wiremock'
     testCompile group: 'org.assertj', name: 'assertj-core'
 }
+
+shadowJar {
+    mergeServiceFiles()
+    classifier ''
+
+    relocate('com.palantir.remoting2.errors', 'com.palantir.atlasdb.http.errors')
```

Comment:
Packages are usually shaded to a different namespace but they retain their original package name. For example:
```
relocate('com.palantir.remoting2.errors', 'com.palantir.atlasdb.shaded.com.palantir.remoting2.errors')
```